### PR TITLE
feature/vocabularies Added IRI for all referenced ontologies

### PR DESCRIPTION
**Pull request by NSeydoux**
Related to #21 

All the vocabularies are now associated with their IRI, and there is a first draft of highlight for RDF and RDFS.

### CODE REVIEW

File: _posts/learn/build-on-solid/vocabularies/2019-02-02-core.md [hunk 1]
```diff
@@ -11,6 +11,37 @@ The following vocabularies are essential, because they describe... vocabularies.
...
 # <a id='rdf'/> RDF
 
+## Prefix and namespace
+- rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+
+## Description
+
+RDF and RDFS are the foundation on which all other vocabularies are built. They enable the description of a graph of [resources](rdfs:Resource), each identified by an IRI.
+
+RDF graphs are based on the notion of [statement](rdf:Statement): a statement can be compared to a very simple sentence <subject, verb, complement>, e.g. `Charlie is a Person`. In the RDF terminology, we say that a [statement](rdf:Statement) has an [subject](rdf:subject) (the resource the statement is about), a [property](rdf:property) (the verb of the sentence, defining a characteristic of the subject) and an [object](rdf:object) (the value for the property for the subject). Therefore, the sentence `Charlie is a Person` could be expressed in RDF as `ex:Charlie rdf:type ex:Person`, where `<http://example.org/ns#Charlie>` is the subject, `rdf:type` the property, and `<http://example.org/ns#Person>` the object. Subject and object are necessarily individuals identified with an IRI, while the object is either an individual with an IRI, or a literal with a value (e.g. string or int).
```
Comment (Vinnl) [inline _posts/learn/build-on-solid/vocabularies/2019-02-02-core.md]: I think you mean "Subject and _predicate_ are necessarily...identified with an IRI"?

File: _posts/learn/build-on-solid/vocabularies/2019-02-03-solid.md [hunk 2]
```diff
@@ -9,4 +9,47 @@ exclude: true
...
+Describes terms for POSIX-like files and directory listings. Used in the [Solid REST API](https://github.com/solid/solid-spec#https-rest-api) when listing Containers.
+
 # <a id="ui"/> UI and forms
+
+## Prefix and namespace
+- ui: http://www.w3.org/ns/ui#
+
+## Description
+
+The User Interface ontology allows the definition of forms for processing RDF model data, and include a bootstrap form for editing forms. It allows user interface hints such as background colors, can be associated with objects and classes.
```
Comment (Vinnl) [inline _posts/learn/build-on-solid/vocabularies/2019-02-03-solid.md]: I think Tim mentioned that this is a rather incomplete and ad-hoc vocabulary, and even had different versions available - so we might want to warn a bit against using it?

Then again, I'm not sure of the "official" status of the others, so maybe just let readers decide for themselves whether they need this.

File: _posts/learn/build-on-solid/vocabularies/2019-02-02-core.md (hunk 1 above)
Comment (NSeydoux) [reply _posts/learn/build-on-solid/vocabularies/2019-02-02-core.md]: Indeed

File: _posts/learn/build-on-solid/vocabularies/2019-02-03-solid.md (hunk 2 above)
Comment (NSeydoux) [reply _posts/learn/build-on-solid/vocabularies/2019-02-03-solid.md]: I think you're right:  featuring these vocabularies on this page gives them a certain "aura" of official recommendation which could be misleading. I'll add a "status" label to each, to identify the stable ones from the more experimental

**Pull request merged**